WORKSPACE

```diff
@@ -1,5 +1,7 @@
 load("//tf_dependency:tf_configure.bzl", "tf_configure")
+load("//external/gpu:cuda_configure.bzl", "cuda_configure")
```

Comment:
@seanpmorgan Sean, perhaps we could move tf_dependency to external directory as well, any idea?

Reply:
I agree they should both live in a sub-directory. How would you feel about `build_deps`?

Reply:
Would you prefer we move tf_dependency into `build_deps` in a separate PR?

Reply:
totally agree